crates/api/src/site/mod_log.rs

```diff
@@ -20,6 +22,21 @@ use lemmy_db_views_moderator::structs::{
 use lemmy_utils::{error::LemmyError, ConnectionId};
 use lemmy_websocket::LemmyContext;
 
+enum ModlogEnum {
```

Comment:
You can drop the `Mod` from the start of each of these variants and also rename the enum to `Modlog`.

Reply:
This is specifically matching the same enum declaration on the frontend side. Can we change back to match?